.buildobeginsrc

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
The repository doesn't need the configuration file. It should be used in a custom project directory. So, for example:

1. you install buildo begins (npm install buildo-begins -g)
2. you create a custom directory
3. there, you derive your pem file and create a .buildobeginsrc config file